test/sqllogictest/transform/join_fusion.slt

```diff
@@ -194,21 +194,27 @@ EXPLAIN WITH(arity, join_impls) SELECT l_partkey AS col24843 , l_orderkey AS col
   AND l_extendedprice = MOD (o_totalprice , 5 ) ;
 ----
 Explained Query:
-  Project (#1, #0, #1) // { arity: 3 }
-    Join on=(#2 = #4 AND #3 = #5) type=differential // { arity: 6 }
-      implementation
-        %1:orders[#0, #1]KKef » %0:lineitem[#2, #3]KKef » %2:customer[×]ef
-      ArrangeBy keys=[[#2, #3]] // { arity: 4 }
-        Project (#0, #1, #4, #6) // { arity: 4 }
-          Filter (#4 = (#4 % 5)) // { arity: 8 }
-            Get materialize.public.lineitem // { arity: 8 }
-      ArrangeBy keys=[[#0, #1]] // { arity: 2 }
-        Project (#2, #3) // { arity: 2 }
-          Filter (#2 = (#2 % 5)) // { arity: 5 }
-            ReadExistingIndex materialize.public.orders lookup_value=(134) // { arity: 5 }
-      ArrangeBy keys=[[]] // { arity: 0 }
-        Project () // { arity: 0 }
-          ReadExistingIndex materialize.public.customer lookup_value=(134) // { arity: 4 }
+  Return // { arity: 3 }
+    Project (#1, #0, #1) // { arity: 3 }
+      Join on=(#2 = #4 AND #3 = #5) type=differential // { arity: 6 }
+        implementation
+          %1:orders[#0, #1]KKef » %0:lineitem[#2, #3]KKef » %2:customer[×]ef
+        ArrangeBy keys=[[#2, #3]] // { arity: 4 }
+          Project (#0, #1, #4, #6) // { arity: 4 }
+            Filter (#4 = (#4 % 5)) // { arity: 8 }
+              Get materialize.public.lineitem // { arity: 8 }
+        ArrangeBy keys=[[#0, #1]] // { arity: 2 }
+          Project (#2, #3) // { arity: 2 }
+            Filter (#2 = (#2 % 5)) // { arity: 5 }
+              ReadExistingIndex materialize.public.orders lookup_value=(134) // { arity: 5 }
+        ArrangeBy keys=[[]] // { arity: 0 }
+          Project () // { arity: 0 }
+            ReadExistingIndex materialize.public.customer lookup_value=(134) // { arity: 4 }
+  With
+    cte l0 =
```

Comment:
This is a bit weird: `l0` is defined as a binding, but is not referenced anywhere (I think this is because it's hidden by the `ReadExistingIndex` rendering).

Can you change the rendering code to print `lookup_value=$local_id` instead of `lookup_value=($vals)` if the join input is a `Get $local_id` instead of `ArrangeBy` of a `Constant`?

Reply:
Done. It's printing it like this now: `lookup_values=<Get l0>`